selenium/qa/src/test/java/pages/SignUpPage.java

```diff
@@ -70,6 +70,7 @@ public void createAccount(String firstName, String lastName, String emailAddress
 	}
 
 	public void navigate() {
+		navigate();
```

Comment:
Won't this call itself?
```suggestion
		super.navigate();
```

Reply:
actually this method is not used, that is why i didn't noticed it, i will remove it.